lib/trento/application/usecases/sap_systems/dto/health_summary_dto.ex

```diff
@@ -3,7 +3,14 @@ defmodule Trento.Application.UseCases.SapSystems.HealthSummaryDto do
   HealthSummary for SAP Systems
   """
 
-  @required_fields :all
+  @required_fields [
```

Comment:
We could even make the `database_id` mandatory, because having a database is hard requirement to have a sap system. But well, to play it safe we can remove it hehe